scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -589,10 +592,10 @@ def handle(self):
             with open(d_log, "w") as dlog_fp:
                 dlog_fp.write(stderr.decode())
 
-        except subprocess.CalledProcessError:
+        except subprocess.CalledProcessError as call_error:
             halt_monitor_evt.set()
             self.instance.status = "error"
-            self.instance.reason = "Device issue (Flash error)"
+            self.instance.reason = f'Unable to open flashing process: {call_error.output.decode(errors="ignore")}'
```

Comment:
i'd suggest to keep the old `Device issue (Flash error` + add more details from the exception